packages/web/src/scss/components/Grid/README.md

```diff
@@ -58,6 +58,62 @@ Responsive columns:
 12-column guides. This behavior is intentional as the 5-column grid serves to
 equally distribute 5 items in a row.
 
+## Custom Spacing
+
+Use CSS custom properties to define custom gutters between items. Set the `--grid-spacing-y` or the `--grid-spacing-x`
+property to one of spacing token values defined on the `:root` element, e.g. `--grid-spacing-y: var(--spirit-space-800)`.
+This will set the spacing between rows to `var(--spirit-space-800)` for all breakpoints.
```

Comment:
> for all breakpoints

I'm afraid this is not true. Apparently, the default value is used for tablet and desktop:

<img width="519" alt="obrazek" src="https://github.com/user-attachments/assets/addf720a-9d4a-482e-ae8d-f416b833ae51">
<img width="519" alt="obrazek" src="https://github.com/user-attachments/assets/c2985f7d-07ea-44b8-bf8d-243d7f498993">

Looking into the `Stack` implementation, it seems one fallback piece is missing here in `Grid`:

<img width="517" alt="obrazek" src="https://github.com/user-attachments/assets/b08ba00c-80f3-40c2-895c-9b4607c54e5b">